lib/ledger_sync/adaptors/netsuite/adaptor.rb

```diff
@@ -60,6 +60,14 @@ def get(**keywords)
           request(keywords.merge(method: :get))
         end
 
+        def ledger_resource_path(args = {})
+          resource = args.fetch(:resource, nil)
+
+          ret = resource.class.resource_type.to_s # This can be turned into a case statement if we need to override
```

Comment:
How will we deal with `ledger_resource_type_overrides`?

Reply:
For now, we are going to have to just override here as needed with a case. The issue is that the current overrides only considers the class.  But the NS item for example will switch depending on a value of the item.

I'd say lave and then we perhaps combine concepts as we figure out exactly how to handle the more polymorphic relationships.